.github/workflows/pre-commit.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     env:
       # these checks currently require a ridiculous number of dependencies :)
       # consider repackaging them in a container in the future
-      SKIP: "web-js-lint-check,web-css-lint-check,web-misc-lint-check"
+      SKIP: "web-js-lint-check,web-css-lint-check,web-misc-lint-check,black"
```

Comment:
Is the black check particularly slow, or can we actually just leave it in there?

Reply:
this is to avoid introducing a CI check for which this pr doesn't add a solution which doesn't rely on our optional pre-commit